modules/redpanda/redpanda_test.go

```diff
@@ -179,6 +179,112 @@ func TestRedpandaWithAuthentication(t *testing.T) {
 	}
 }
 
+func TestRedpandaWithBootstrapUserAuthentication(t *testing.T) {
+	ctx := context.Background()
+	// redpandaCreateContainer {
+	ctr, err := redpanda.Run(ctx,
+		"docker.redpanda.com/redpandadata/redpanda:v23.3.3",
+		redpanda.WithEnableSASL(),
+		redpanda.WithEnableKafkaAuthorization(),
+		redpanda.WithEnableWasmTransform(),
+		redpanda.WithNewServiceAccount("superuser-1", "test"),
+		redpanda.WithNewServiceAccount("superuser-2", "test"),
+		redpanda.WithNewServiceAccount("no-superuser", "test"),
+		redpanda.WithSuperusers("superuser-1", "superuser-2"),
+		redpanda.WithEnableSchemaRegistryHTTPBasicAuth(),
+		redpanda.WithEnableAdminAPIAuthentication(),
+	)
+	testcontainers.CleanupContainer(t, ctr)
+	require.NoError(t, err)
+	// }
+
+	// kafkaSeedBroker {
+	seedBroker, err := ctr.KafkaSeedBroker(ctx)
+	// }
```

Comment:
I think we are propagating those docs comments, which are needed just once. Let's remove them unless they have a unique name and used in the docs.

```suggestion
	seedBroker, err := ctr.KafkaSeedBroker(ctx)
```

Reply:
I believe I fixed this comment usage throughout the tests.